src/web-ui/src/repositories/usersRepository.js

```diff
@@ -45,5 +45,14 @@ export default {
         if (!user)
             throw "user required"
         return connection.put(`${resource}/id/${user.id}`, user)
+    },
+    verifyAndUpdateUserPhoneNumber(userID, phonenumber) {
+        if (!userID || userID.length == 0)
+            throw "userID required"
+        let payload = {
+            userid: userID,
+            phonenumber: phonenumber        
```

Comment:
Sync these changes with requested naming changes earlier in PR (i.e. snake case).